### PR TITLE
start remote registry as unprivileged user in reg.py

### PR DESCRIPTION
**Pull request by dadevel**
This PR implements the trick discovered by @antonioCoco in https://twitter.com/splinter_code/status/1715876413474025704 to trigger the start of the RemoteRegistry service as unprivileged user by opening the winreg named pipe over SMB.
Additionally `HKEY_USERS` can now be accessed trough `reg.py`.

demo

### CODE REVIEW

File: examples/reg.py [hunk 1]
```diff
@@ -200,6 +201,17 @@ def run(self, remoteName, remoteHost):
             if self.__remoteOps:
                 self.__remoteOps.finish()
 
+    def triggerWinReg(self):
+        # original idea from https://twitter.com/splinter_code/status/1715876413474025704
+        tid = self.__smbConnection.connectTree('IPC$')
+        try:
+            self.__smbConnection.openFile(tid, r'\winreg', 0x12019f, creationOption=0x40, fileAttributes=0x80)
```
Comment (dadevel) [inline examples/reg.py:208]: No clue what the hex values do, just copied them over from `psexec.py`.

File: examples/reg.py [hunk 2]
```diff
@@ -413,8 +425,10 @@ def __strip_root_key(self, dce, keyName):
             raise Exception('Error parsing keyName %s' % keyName)
         if rootKey.upper() == 'HKLM':
             ans = rrp.hOpenLocalMachine(dce)
-        elif rootKey.upper() == 'HKU':
+        elif rootKey.upper() == 'HKCU':
```
Comment (anadrianmanrique) [inline examples/reg.py:428]: nice to have this fixed. please update help in line 551, as it's referencing root key names, and it's not currently contemplating HKCU

**Comment by anadrianmanrique** [comment]
@dadevel thanks for your PR. I'm testing it, so fat it's working as expected. Please apply the suggested changes in the review in order to merge these changes in the main branch. Thanks

**Comment by dadevel** [comment]
Hi @anadrianmanrique, thanks for reviewing my PR. I updated the help message.

**Pull request merged**